### PR TITLE
Fixing third-party cluster drivers not updating

### PR DESCRIPTION
**Pull request by nathan-jenan-rancher**
This change corrects an issue where third-party drivers were not updating
their schemas after the driver binary was updated (via a url change). This
change corrects that so that the schema will update.

Issue:
#17712 

Depends on:
https://github.com/rancher/kontainer-engine-driver-example/pull/8

### CODE REVIEW

File: tests/core/test_kontainer_drivers.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 import pytest
+import sys
 
 from rancher import ApiError
 from .conftest import wait_for_condition, wait_until
 
+# FIXME update this to an actual release of kontainer engine example
+#  driver
+NEW_DRIVER_URL = "https://github.com/nathan-jenan-rancher/kontainer-" \
+                 "engine-example-driver/releases/download/v2.2.0/" \
```
Comment (cjellick) [inline tests/core/test_kontainer_drivers.py]: wrong version. you want `v0.2.2` not `2.2.0`

Comment (cjellick) [reply tests/core/test_kontainer_drivers.py]: @nathan-jenan-rancher i did the release https://github.com/rancher/kontainer-engine-driver-example/releases/tag/v0.2.1

Comment (nathan-jenan-rancher) [reply tests/core/test_kontainer_drivers.py]: I will update the url.

**Comment by erikwilson** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**